src/client/routes/Portfolio/index.tsx

```diff
@@ -24,124 +28,154 @@ const HeaderCard = styled(SummaryCard)`
+
+  ${({ active, theme }) =>
+    active &&
+    `
+    background-color: ${theme.colors.backgroundVariant};
+    color: ${theme.colors.titlesVariant};
+    border-color: ${theme.colors.primary};
+  `}
 `;
 
-const StyledSummaryCard = styled(SummaryCard)`
-  width: 100%;
-  grid-column: 1 / 3;
-  ${halfWidthCss};
+const StyledSliderCard = styled(SliderCard)`
+  padding: 3rem;
+  margin: 0;
 `;
 
-const StyledSpinnerLoading = styled(SpinnerLoading)`
+const StyledBorrowerContainer = styled.div`
   grid-column: 1 / 3;
-  flex: 1;
-  margin: 10rem 0;
 `;
 
 export const Portfolio = () => {
   const { t } = useAppTranslation(['common', 'home']);
-  const { NETWORK_SETTINGS } = getConfig();
   const isMounting = useIsMounting();
+  const location = useLocation();
+  const dispatch = useAppDispatch();
+  const history = useHistory();
   const walletIsConnected = useAppSelector(WalletSelectors.selectWalletIsConnected);
 
-  const currentNetwork = useAppSelector(NetworkSelectors.selectCurrentNetwork);
-  const currentNetworkSettings = NETWORK_SETTINGS[currentNetwork];
-  const vaultsSummary = useAppSelector(VaultsSelectors.selectSummaryData);
   // const labsSummary = useAppSelector(LabsSelectors.selectSummaryData);
-  const walletSummary = useAppSelector(TokensSelectors.selectSummaryData);
   const userTokens = useAppSelector(TokensSelectors.selectUserTokens);
   const activeModal = useAppSelector(ModalSelectors.selectActiveModal);
   const appStatus = useAppSelector(AppSelectors.selectAppStatus);
   const tokensListStatus = useAppSelector(TokensSelectors.selectWalletTokensStatus);
   const generalLoading = (appStatus.loading || tokensListStatus.loading || isMounting) && !activeModal;
-  const userTokensLoading = generalLoading && !userTokens.length;
+  const userAddress: string | undefined = location.pathname.split('/')[2];
+  const userPortfolio = useAppSelector(LinesSelectors.selectUserPortfolio);
 
-  const netWorth = toBN(vaultsSummary.totalDeposits)
-    .plus(walletSummary.totalBalance)
-    // .plus(labsSummary.totalDeposits)
-    .toString();
-
-  const summaryCardItems = [
-    { header: t('dashboard.total-net-worth'), Component: <Amount value={netWorth} input="usdc" /> },
-  ];
-  if (walletIsConnected) {
-    summaryCardItems.push({
-      header: t('dashboard.available-deposit'),
-      Component: <Amount value={walletSummary.totalBalance} input="usdc" />,
-    });
-  }
-  if (currentNetworkSettings.earningsEnabled) {
-    summaryCardItems.push(
-      {
-        header: t('dashboard.vaults-earnings'),
-        Component: <Amount value={vaultsSummary.totalEarnings} input="usdc" />,
-      },
-      {
-        header: t('dashboard.vaults-est-yearly-yield'),
-        Component: <Amount value={vaultsSummary.estYearlyYeild} input="usdc" />,
+  const userTokensLoading = generalLoading && !userTokens.length;
+  const [currentRole, setRole] = useState<string>(BORROWER_POSITION_ROLE);
+  const [data, setdata] = useState<any[]>([]);
```

Comment:
typo

Reply:
also please add types not so we know what data actually means

Reply:
@Seroxdesign Are you taking this?